src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -172,25 +173,33 @@ public override Task StartDefaultOsBrowserAsync(string url)
                     throw new MsalClientException(MsalError.LinuxXdgOpen, MsalErrorMessage.LinuxOpenAsSudoNotSupported);

```

Comment:
Can you check if you can start edge on Linux with `sudo` ? This exception prevents doing `sudo xdg-open` but maybe Edge is ok?

Reply:
No edge does not open with sudo either